Firmware/ultralcd.cpp

```diff
@@ -3246,33 +3245,30 @@ bool lcd_wait_for_pinda(float temp) {
 		lcd_print(ftostr3(current_temperature_pinda));
 		lcd_print('/');
 		lcd_print(ftostr3(temp));
-		lcd_print(LCD_STR_DEGREE);
+		lcd_print(LCD_STR_DEGREE[0]);
```

Comment:
:+1: even though it is not completely obvious at the first look, forcing a call to `lcd_print` with just a character argument (instead of an array of chars = string) makes the compiler generate better code.